src/styles/index.scss

```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import '/src/styles/blocks/variables';
```

Comment:
Ensure that the import path for the variables file is correct and relative to the SCSS file location. The current path '/src/styles/blocks/variables' should be './blocks/variables' to align with the project structure.